app/templates/components/donations/create-donation.hbs

```diff
@@ -1,11 +1,10 @@
+<h3>Support this project each month</h3>
 <div class="amounts">
 {{#each presetAmounts as |presetAmount|}}
   {{donations/donation-amount-button
-    customAmount=customAmount
     presetAmount=presetAmount
-    selectedAmount=amount
-    setCustomAmount=(action (mut customAmount))
-    setAmount=(action (mut amount))
+    click=(action (mut amount) presetAmount)
```

Comment:
There was a click handler inside the child component, which triggers 2 actions on the parent. This removes that behavior and simply puts the parent in charge of the click. On click, `amount` will be set to the value of `presetAmount`